test/modules/core/shaderlib/project/project-32-64-glsl.spec.js

```diff
@@ -19,12 +19,11 @@
 // THE SOFTWARE.
 
 import test from 'tape-promise/tape';
-import assert from 'assert';
 import GL from '@luma.gl/constants';
 
 // import {COORDINATE_SYSTEM, Viewport, WebMercatorViewport} from 'deck.gl';
 import {COORDINATE_SYSTEM, WebMercatorViewport} from 'deck.gl';
-import {project, project32} from '@deck.gl/core';
+import {project, project32, log} from '@deck.gl/core';
```

Comment:
This is just a test file, I don't think it's a big deal. `assert` is such a trivial utility that I feel exporting it is an overkill...

Reply:
Yesh... though I do think it is ugly and with time this may be copy pasted into other files. 

Just paste a copy of the assert function into this file or replace with if/throw?

TBH, I would prefer to remove `assert` from the Log class, it is a bit of an overreach.